cypress/integration/search/date_filter_spec.js

```diff
@@ -234,8 +237,8 @@ describe('SF15699 Search Date Filter', () => {
             searchAndValidate(`before:${secondDateEarly.query} in:town-square ${timestamp}`, [firstMessage]);
         });
 
-        it('can be used in conjunction with "from:"', function() {
```

Comment:
Were these causing problems?

Reply:
Actually the issue is `this.newAdmin` being undefined.